miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridge.kt

```diff
@@ -92,28 +96,43 @@ internal class UserInfoBridge {
 
     private fun onHasAccessTokenPermission(callbackObj: CallbackObj) {
         val tokenPermission = parseAccessTokenPermission(callbackObj)
+        if (doesMatchManifest(tokenPermission)) {
 
-        val successCallback = { accessToken: TokenData ->
-            bridgeExecutor.postValue(callbackObj.id, Gson().toJson(accessToken))
-        }
-        val errorCallback = { message: String ->
-            bridgeExecutor.postError(callbackObj.id, "$ERR_GET_ACCESS_TOKEN $message")
-        }
+            val successCallback = { accessToken: TokenData ->
+                accessToken.accessTokenScope = tokenPermission
+                bridgeExecutor.postValue(callbackObj.id, Gson().toJson(accessToken))
+            }
+            val errorCallback = { message: String ->
+                bridgeExecutor.postError(callbackObj.id, "$ERR_GET_ACCESS_TOKEN $message")
+            }
 
-        try {
-            userInfoBridgeDispatcher.getAccessToken(miniAppId, tokenPermission, successCallback, errorCallback)
-        } catch (e: MiniAppSdkException) {
-            if (e.message == "The `UserInfoBridgeDispatcher.getAccessToken` $NO_IMPL")
-                userInfoBridgeDispatcher.getAccessToken(miniAppId, successCallback, errorCallback)
-            else
-                throw e
+            try {
+                userInfoBridgeDispatcher.getAccessToken(miniAppId, tokenPermission, successCallback, errorCallback)
+            } catch (e: MiniAppSdkException) {
+                if (e.message == "The `UserInfoBridgeDispatcher.getAccessToken` $NO_IMPL")
+                    userInfoBridgeDispatcher.getAccessToken(miniAppId, successCallback, errorCallback)
+                else
+                    throw e
```

Comment:
I am not sure, if we can get any other `MiniAppSdkException` message except `$NO_IMPL` one inside this `try {...} catch` block.  So, maybe we don't need to `throw e`.

Reply:
there is still chance that there is another exception, that's why I also have a test case for this.